addons/mail/static/src/models/message_action_list.js

```diff
@@ -1,186 +1,195 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { attr, one } from '@mail/model/model_field';
-import { clear, insertAndReplace } from '@mail/model/model_field_command';
-import { markEventHandled } from '@mail/utils/utils';
+import { attr, many, one } from '@mail/model/model_field';
+import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'MessageActionList',
     recordMethods: {
         /**
          * @private
-         * @param {MouseEvent} ev
+         * @returns {integer}
          */
-        onClick(ev) {
-            markEventHandled(ev, 'MessageActionList.Click');
+        _computeActionsWithoutCompactCount() {
+            return this.messageActions.filter((action) => !action.messageActionListOwnerAsToggleCompact).length;
```

Comment:
`.filter()` is `O(n)` complexity, so theoretically could have better performance by instead:
- define computed one field on `MessageAction/isNonCompactActionContribution`, which is 1 for all except toggle compact, and then 0 for the toggle compact button;
- replace `compute` here by `sum: messageActions. isNonCompactActionContribution `;

Reply:
Why resolve? This comment is still applicable?